webpack/ForemanTasks/Components/TasksTable/TasksTableHelpers.js

```diff
@@ -24,3 +28,39 @@ export const addSearchToURL = (path, query) => {
   url.addSearch({ ...query, include_permissions: true });
   return url.toString();
 };
+
+export const getDuration = (start, finish) => {
+  if (!start && !finish)
+    return { text: __('N/A'), tooltip: __('No start or end dates') };
+
+  if (!start && finish) {
+    return { text: __('N/A'), tooltip: __('Task was canceled') };
+  }
+
+  const dateOptions = {
+    largest: 1,
+    language: documentLocale(),
+    fallbacks: ['en'],
+    round: true,
+  };
+
+  start = new Date(isoCompatibleDate(start));
+  if (start && !finish) {
+    finish = new Date();
+    const duration = finish - start;
+    return {
+      text: `${humanizeDuration(duration, dateOptions)} +`,
+      tooltip: `${__('More than')} ${numberWithDelimiter(
+        (finish - start) / 1000
+      )} ${__('seconds')}`,
```

Comment:
![Screenshot from 2019-11-04 15-21-43](https://user-images.githubusercontent.com/26363699/68124102-3335b800-ff17-11e9-8b96-64041b66a3b6.png)

I think that when it comes to minutes, the user doesn't need to know how many seconds are in those minutes.

Reply:
Why should it be different then what is in the `Task Raw` page? 
[previous ruby code](https://github.com/MariaAga/foreman-tasks/blob/90a49ddfaae7fb27749939d49371b42b6a2a43d2/app/helpers/foreman_tasks/foreman_tasks_helper.rb#L46)

Reply:
@MariaAga I think that when moving a page to React we should always think on how to improve it, or if it deserves a redesign.